Property POSITION_INDEPENDENT_CODE does not match the INTERFACE_POSITION_INDEPENDENT_CODE property

We get this following [error](https://circleci.com/gh/STEllAR-GROUP/phylanx/34669) using HighFive
```console
-- Configuring done
CMake Error: Property POSITION_INDEPENDENT_CODE on target "fileio_primitive" does
not match the INTERFACE_POSITION_INDEPENDENT_CODE property requirement
of dependency "HighFive".
```
Isn't HighFive a header-only library? Would you please take a look at new changes and help us with a solution?

--- a/CMake/HighFiveConfig.cmake.in
+++ b/CMake/HighFiveConfig.cmake.in
@@ -1,12 +1,10 @@
 function(copy_interface_properties target source)
   foreach(prop
-          INTERFACE_AUTOUIC_OPTIONS
           INTERFACE_COMPILE_DEFINITIONS
           INTERFACE_COMPILE_FEATURES
           INTERFACE_COMPILE_OPTIONS
           INTERFACE_INCLUDE_DIRECTORIES
           INTERFACE_LINK_LIBRARIES
-          INTERFACE_POSITION_INDEPENDENT_CODE
           INTERFACE_SOURCES
           INTERFACE_SYSTEM_INCLUDE_DIRECTORIES)
     set_property(TARGET ${target} APPEND PROPERTY ${prop} $<TARGET_PROPERTY:${source},${prop}>)

--- a/tests/test_library/CMakeLists.txt
+++ b/tests/test_library/CMakeLists.txt
@@ -24,5 +24,15 @@ target_include_directories(simpleton
     $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
     $<INSTALL_INTERFACE:include>)
 target_link_libraries(simpleton PUBLIC HighFive)
-install(TARGETS simpleton EXPORT simpletonTarget ARCHIVE DESTINATION lib)
+set_property(TARGET simpleton PROPERTY POSITION_INDEPENDENT_CODE ON)
+
+add_library(otherton "src/simpleton.cpp")
+target_include_directories(otherton
+    PUBLIC
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+    $<INSTALL_INTERFACE:include>)
+target_link_libraries(otherton PUBLIC HighFive)
+set_property(TARGET otherton PROPERTY POSITION_INDEPENDENT_CODE OFF)
+
+install(TARGETS simpleton otherton EXPORT simpletonTarget ARCHIVE DESTINATION lib)
 install(EXPORT simpletonTarget DESTINATION lib)

